stories/basics/WVList.stories.tsx

```diff
@@ -181,3 +181,72 @@ export const InfiniteScrolling: StoryObj = {
     );
   },
 };
+
+const RestorableList = ({ id }: { id: string }) => {
+  const cacheKey = "window-list-cache-" + id;
+
+  const ref = useRef<WVListHandle>(null);
+
+  const [offset, cache] = useMemo(() => {
+    const serialized = sessionStorage.getItem(cacheKey);
+    if (!serialized) return [];
+    return JSON.parse(serialized) as [number, CacheSnapshot];
+  }, []);
+
+  useEffect(() => {
+    if (!ref.current) return;
+    const handle = ref.current;
+
+    window.scrollTo(0, offset ?? 0);
+
+    return () => {
+      sessionStorage.setItem(
+        cacheKey,
+        JSON.stringify([window.scrollY, handle.cache])
```

Comment:
> In this case, it could be more appropriate to get the offset by exposing the scrollOffset using WVListHandle instead of using window.scrollY (or scrollX). In the case of storybook, when you hide WVList, window.scrollY will return 0 in useEffect cleanup, but scrollOffset allows you to retrieve the scroll offset stored in VirtualStore. 

I think returning scrollOffset from WVListHandle will not be enough in this case also. The scrollOffset is window.scrollY - offset to the window top so you can't restore scroll position from it.
So my current recommendation is just keep previous window.scrollY by yourself. It may not be clean but it works.